packages/core/test/acceptance/application.acceptance.ts

```diff
@@ -127,34 +96,10 @@ describe('Bootstrapping the application', () => {
         }
       }
       const app = new Application({
-        components: [ConfigComponent, GreetingComponent, RestComponent],
+        components: [ConfigComponent, GreetingComponent],
       });
-      const server = app.getServerSync(RestServer);
 
-      expect(server.getSync('greeting')).to.equal('Hi!');
+      expect(app.getSync('greeting')).to.equal('Hi!');
     });
   });
 });
-
-describe('Starting the application', () => {
-  it('starts an HTTP server', async () => {
```

Comment:
We already have a test in core/test/unit/application.test.ts that validates the idea of starting an application and that the server instances have also started.

The full integration/e2e test case is covered by codehub-example, isn't it?

Reply:
> We already have a test in core/test/unit/application.test.ts that validates the idea of starting an application and that the server instances have also started.

Good point, I think that's enough 👍 

> The full integration/e2e test case is covered by codehub-example, isn't it?

I think we should not treat codehub-example as a replacement for e2e tests. codehub-example started as an attempt to write a "real world" application and use this exercise to drive the API of loopback-next. We never really finished this app, it may be actually better to remove it, so that we don't confuse community users with a half-done app. I am not sure if the codehub app was updated to follow the latest conventions.